### PR TITLE
EZP-28890: Updated translations sync procedure

### PR DESCRIPTION
**Pull request by bdunogier**
> Story: [EZP-28890](https://jira.ez.no/browse/EZP-28890)
 
Moved the translation script to ezsystems/ezplatform-i18n, as it requires this package.

### TODO
- [x] Figure out if we only add the EE packages to the EE meta repo, or if we keep a unique script with conditions. No personal preference.
- [ ] Merge https://github.com/ezsystems/ezplatform-i18n/pull/37
- [x] Squash

### CODE REVIEW

File: bin/synchronize-translations.sh [hunk 1]
```diff
@@ -4,18 +4,41 @@
...
+
+if [ -d "./vendor/ezsystems/ezstudio-notifications" ]; then
+  rm ./vendor/ezsystems/ezplatform-i18n/ezstudio-notifications/*.xlf
+  cp ./vendor/ezsystems/ezstudio-notifications/bundle/Resources/translations/*.xlf ./vendor/ezsystems/ezplatform-i18n/ezstudio-notifications
+fi
+
+echo '# Strip english locale suffix from filename';
+rename 's/\.en\./\./g' ./vendor/ezsystems/ezplatform-i18n/*/*
+
+echo 'Translation synchronization done !';
```
Comment (alongosz) [inline bin/synchronize-translations.sh]: Nitpick: missing new line before EOF

**Comment by bdunogier** [comment]
Ping @alongosz & @lserwatka. The PR has completely changed. Please see https://github.com/ezsystems/ezplatform-i18n/pull/37 for the follow-up. It should answer your concern (that I share) to some extent, @alongosz. At least, it moves it to another place :)

**Comment by lserwatka** [comment]
@bdunogier you have some GraphQL changes in this PR?

**Comment by bdunogier** [comment]
As my son would say: "oops". Thank you :)

**Comment by bdunogier** [comment]
Yeah, "oops". There was actually nothing to do. I guess I got distracted.

**Pull request merged**